ROS/controls/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>controls</name>
+  <version>0.0.0</version>
+  <description>The controls package</description>
+  
+  <maintainer email="lvov.g@northeastern.edu">Gary Lvov</maintainer>
+  <author email="pirtle.m@northeastern.edu">Maxwell Pirtle</author>
+
+  <license>TODO</license>
```

Comment:
We should have a discussion what open source license to use for this project that should be universal across the board, perhaps we can discuss this at the next meeting